src/code/z_en_item00.c

```diff
@@ -1,10 +1,23 @@
-#include "global.h"
-#include "attributes.h"
+#include "z_en_item00.h"
+#include "overlays/actors/ovl_En_Elf/z_en_elf.h"
 #include "overlays/actors/ovl_En_Elforg/z_en_elforg.h"
+
+#include "libc64/qrand.h"
+#include "attributes.h"
+#include "gfx.h"
+#include "gfx_setupdl.h"
+#include "ichain.h"
+#include "macros.h"
+#include "rand.h"
+#include "sfx.h"
+#include "sys_matrix.h"
+#include "z64effect.h"
+#include "z64lib.h"
+#include "z64play.h"
+
+#include "global.h"
```

Comment:
It feels kinda dumb adding all those individual headers and then add `global.h` too.
Why not just leave alone those files that import `gameplay_keep.h` up until we have the new assets system?

Reply:
Since the work is already done, I would kind of say just leave it for now. It does make sense for future PRs though to wait.